app-emulation/vagrant/vagrant-2.2.6.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+USE_RUBY="ruby24 ruby25 ruby26"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="vagrant.gemspec"
+RUBY_FAKEGEM_EXTRAINSTALL="keys plugins templates version.txt"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit bash-completion-r1 ruby-fakegem
+
+DESCRIPTION="A tool for building and distributing development environments"
+HOMEPAGE="https://vagrantup.com/"
+SRC_URI="https://github.com/hashicorp/vagrant/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+virtualbox"
+RESTRICT="test"
+
+RDEPEND="${RDEPEND}
+	app-arch/libarchive
+	net-misc/curl
+	virtualbox? ( || ( app-emulation/virtualbox app-emulation/virtualbox-bin ) )"
+
+ruby_add_rdepend "
+	>=dev-ruby/bcrypt_pbkdf-1.0.0
+	>=dev-ruby/childprocess-0.6.0
+	>=dev-ruby/ed25519-1.2.4
+	>=dev-ruby/erubis-2.7.0
+	>=dev-ruby/hashicorp-checkpoint-0.1.5
+	>=dev-ruby/i18n-1.1.1:1
+	>=dev-ruby/listen-3.1.5
+	<dev-ruby/log4r-1.1.11
+	<dev-ruby/mime-types-3:*
+	>=dev-ruby/net-ssh-5.1.0:*
+	>=dev-ruby/net-sftp-2.1
+	>=dev-ruby/net-scp-1.2.0
+	dev-ruby/rest-client:2
+	>=dev-ruby/rubyzip-1.2.2:*
+	>=dev-ruby/vagrant_cloud-2.0.2
```

Comment:
gemspec contains 
` s.add_dependency "vagrant_cloud", "~> 2.0.3"`

please adjust here as well

Reply:
Yes it is done